### DISCUSSION TITLE
Can PriceSchedule validation allow value 0?

### DISCUSSION THREAD

**Discussion by adamturski** [finos/common-domain-model#3413] [Q&A] [answered]
Currently there are some validation rules for PriceSchedule, especially:
PriceSchedulePositiveFXRate
PriceSchedulePositiveSpotRate
PriceSchedulePositiveCashPrice

which do not allow value 0 as a PriceSchedule.value. For example PriceSchedulePositiveCashPrice:
```
if priceExpression -> priceType = PriceTypeEnum -> CashPrice then value > 0
```

For Quantity type there is a validation rule NonNegativeQuantityNonNegativeQuantityAmount which allows 0 as value.

Can such validation rules can be relaxed and allow 0 as a value?

**Comment by eteridvalishvili** [comment]
Hi @adamturski Have you been able to resolve this question? Please let me know if you need to get in touch with the CDM maintainers.

**Comment by aturskifragmoschain** [comment]
@eteridvalishvili I resolved that by overriding the PriceSchedulePositiveCashPrice with my own implementation which allows 0. The ticket here is more a question if such relaxed validation can be provided in next releases.

**Comment by eteridvalishvili** [comment]
Hi @finos/cdm-maintainers Are you able to help resolve this question?

**Comment by eteridvalishvili** [comment]
Hi @adamturski  I hope this message finds you well. I'm following up on your question and wanted to check if it's okay to close now and if you've received your answer. If haven’t and if you'd like to keep it open please let me know. 

If you're interested in joining the Contribution Review Working Group (CRWG) meetings, we have sessions scheduled for tomorrow, February 6th at 10 am EST/3 pm GMT, and another on February 20th at the same time. These meetings provide an opportunity to discuss your questions directly with the maintainers. Please let me know your preferences or if you have any further questions or concerns. Looking forward to hearing from you!

**Comment by aturskifragmoschain** [comment, marked as answer]
@eteridvalishvili Let's close the ticket. Like I wrote before, I overrode the rule with my custom logic which is maybe normal as one can customize the CDM rules if necessary.

**Comment by eteridvalishvili** [comment]
Sounds great. Thanks for confirming!
